src/main/cliapp/src/containers/diseaseAnnotationsPage/NewAnnotationForm.js

```diff
@@ -459,488 +458,479 @@ export const NewAnnotationForm = ({
+											options={annotationTypeTerms}
+											editorChange={onDropdownFieldChange}
+											showClear={true}
+											value={newAnnotation.annotationType}
+											className={classNames({ 'p-invalid': submitted && errorMessages.annotationType })}
+										/>
+									</div>
+									<div className={fieldDetailsColumnSize}>
+										<FormErrorMessageComponent errorMessages={errorMessages} errorField={"annotationType"} />
+									</div>
+								</div>
+							</>
+						)}
+
+						{selectedFormFields?.includes("Genetic Modifier Relation") && (
+							<>
+								<div className="grid">
+									<div className={labelColumnSize}>
+										<label htmlFor="diseaseGeneticModifierRelation">Genetic Modifier Relation</label>
+									</div>
+									<div className={widgetColumnSize}>
+										<ControlledVocabularyFormDropdown
+											name="diseaseGeneticModifierRelation"
+											field="diseaseGeneticModifierRelation"
+											options={geneticModifierRelationTerms}
+											editorChange={onDropdownFieldChange}
+											showClear={true}
+											value={newAnnotation.diseaseGeneticModifierRelation}
+											className={classNames({ 'p-invalid': submitted && errorMessages.diseaseGeneticModifierRelation })}
+										/>
+									</div>
+									<div className={fieldDetailsColumnSize}>
+										<FormErrorMessageComponent errorMessages={errorMessages} errorField={"diseaseGeneticModifierRelation"} />
+										<FormErrorMessageComponent errorMessages={uiErrorMessages} errorField={"diseaseGeneticModifierRelation"} />
+									</div>
+								</div>
+							</>
+						)}
+
+						{selectedFormFields?.includes("Genetic Modifiers") && (
+							<>
+								<div className="grid">
+									<div className={labelColumnSize}>
+										<label htmlFor="diseaseGeneticModifier">Genetic Modifiers</label>
+									</div>
+									<div className={widgetColumnSize}>
+										<AutocompleteFormMultiEditor
+											search={geneticModifiersSearch}
+											initialValue={newAnnotation.diseaseGeneticModifiers}
+											fieldName='diseaseGeneticModifiers'
+											name="diseaseGeneticModifiers"
+											subField="modEntityId"
+											valueDisplay={(item, setAutocompleteHoverItem, op, query) =>
+												<SubjectAutocompleteTemplate item={item} setAutocompleteHoverItem={setAutocompleteHoverItem} op={op} query={query} />}
+											onValueChangeHandler={onArrayFieldChange}
+											classNames={classNames({ 'p-invalid': submitted && errorMessages.diseaseGeneticModifiers })}
+										/>
+									</div>
+									<div className={fieldDetailsColumnSize}>
+										<FormErrorMessageComponent errorMessages={errorMessages} errorField={"diseaseGeneticModifiers"} />
+										<GeneticModifiersAdditionalFieldData fieldData={newAnnotation.diseaseGeneticModifiers} />
+									</div>
+								</div>
+							</>
+						)}
+
+						{selectedFormFields?.includes("Internal") && (
```

Comment:
@adamgibs @VarunReddy1111 can these all be turned into components?

Reply:
@oblodgett can change them into components, will change them